### DISCUSSION TITLE
image path ./assets/%!s(<nil>) does not exist

### DISCUSSION THREAD

**Discussion by londi** [kdevo/osprey-delight#40] [Q&A] [answered]
Hi There,
I'm new to hugo and hugo mod and especially to this awesome theme! :)

Sorry if this issue does not meet the correct guideline... but I like to ask you kindly if anyone encountered the same problem and probably found a solution for it.

After a fresh install of hugo and osprey-delight I see the following error after typing "hugo server":
`error image path ./assets/%!s(<nil>) does not exist`

Thanks in advance for any help

londi

EDIT:
I have noticed that there could be a problem on the file "themes/osprey-delight/layouts/partials/sections/gallery/img.html". The previous version of this file (before commit: kdevo 31.10.21) is working.

**Comment by kdevo** [comment, marked as answer]
Hi @londi,

Error message might be a bit cryptic, but the reason likely is that an image in the gallery is missing.
Please ensure that for each gallery content file a valid `image` is specified that you can also find under `./assets` (like [here](https://github.com/kdevo/osprey-delight/blob/e11c568790a20382dcf79278d53df3622ed12246/exampleSite/content/gallery/kdevo.md?plain=1#L5)).

**Comment by kdevo** [comment]
Issue that error message is not clear is tracked under https://github.com/hugo-mods/lazyimg/issues/6
